docs/index.md

```diff
@@ -78,23 +88,23 @@ full list of commands.*
 
 Incorporating these operations into your data lake pipelines provides the same collaboration and organizational benefits you get when managing application code with source control.
 
-### The lakeFS promotion workflow
+### The lakeFS Promotion Workflow
 
 Here's how lakeFS *branches* and *merges* improve the universal process of updating collections with the latest data.
 
-<img src="{{ site.baseurl }}/assets/img/promotion_workflow.png" alt="lakeFS promotion workflow" width="60%" height="60%" />
+<img src="/assets/img/promotion_workflow.png" alt="lakeFS promotion workflow" width="60%" height="60%" />
 
 1. First, create a new **branch** from `main` to instantly generate a complete "copy" of your production data.
 2. Apply changes or make updates on the isolated branch to understand their impact prior to exposure.
 3. And finally, perform a **merge** from the feature branch back to main to atomically promote the updates into production.
 
 Following this pattern, lakeFS facilitates a streamlined data deployment workflow that consistently produces data assets you can have total confidence in.
 
-## What else does lakeFS do?
+## How Can lakeFS Help Me?
```

Comment:
I am missing the dev\test use case. You are mentioning it clearly in the "git-for-data" section but I think it is missing an explicit reference here.

Reply:
added